single_stage_detector/ssd/README.md

```diff
@@ -1,287 +1,27 @@
-# 1. Summary
-Single Shot MultiBox Detector (SSD) is an object detection network. For an input image, the network outputs a set of bounding boxes around the detected objects, along with their classes. For example:
+# Training
 
-![](https://upload.wikimedia.org/wikipedia/commons/3/38/Detected-with-YOLO--Schreibtisch-mit-Objekten.jpg)
 
-SSD is a one-stage detector, both localization and classification are done in a single pass of the network. This allows for a faster inference than region proposal network (RPN) based networks, making it more suited for real time applications like automotive and low power devices like mobile phones. This is also sometimes referred to as being a "single shot" detector for inference.
+## Dataset Download
 
-# 2. Directions
+Install fiftyone. It's recommended to install the package on your host machine, and not inside docker
 
-## Steps to configure machine
-### From Source
-tbd
-### Standard script
-tbd
-### From Docker
-1. Checkout the MLPerf repository
-```
-git clone https://github.com/mlperf/reference.git
-```
-2. Install CUDA and Docker
-```
-source reference/install_cuda_docker.sh
-```
-3. Build the docker image for the single stage detection task
-```
-# Build from Dockerfile
-cd reference/single_stage_detector/
-sudo docker build -t mlperf/single_stage_detector .
+```bash
+pip install fiftyone
 ```
 
-## Steps to download data
+Download the MLPerf subset:
+```bash
+./download_openimages_mlperf.sh -d <DATASET_PATH>
```

Comment:
this is not quite correct as `download_openimages_mlperf.sh` is in the parent folder.

Reply:
The previous README was more of a placeholder and not something we intended to merge. The final version address these issues.